salesagility/SuiteCRM
# Feature Browser Stack

This change contains changes from other pull requests:
#4061
#4058 
#4052 
#4044 
#4049 

## Description



This change adds support for browser stack to the new testing framework. 

## Motivation and Context

- Enables testing across multiple platforms and browsers.

## How To Test This

* See [tests/README.md](https://github.com/daniel-samson/SuiteCRM/blob/feature_cbs_merge/tests/README.md) and configure your environment appropriately
* Run `composer install`
* Run `codecept run install`
* Run `codecept run acceptance`

## Types of changes


### Final checklist
